nucypher/utilities/json_metrics_export.py

```diff
@@ -0,0 +1,64 @@
+from __future__ import unicode_literals
+from prometheus_client.utils import floatToGoString
+from twisted.web.resource import Resource
+from prometheus_client.registry import REGISTRY
+import json
+
+
+class JSONMetricsResource(Resource):
+    """
+    Twisted ``Resource`` that serves metrics in JSON.
+    """
+    isLeaf = True
+
+    def __init__(self, registry=REGISTRY):
+        self.registry = registry
+
+    def render_GET(self, request):
+        request.setHeader(b'Content-Type', "text/json")
+        return self.generate_latest_json()
+
+    @staticmethod
+    def get_sample_labels(labels):
+        if not labels:
```

Comment:
Is this indicating that `labels` is optional or is passing an explicit falsy value part of the API?